yarn.lock

```diff
@@ -16459,7 +16459,7 @@ react-refresh@^0.8.3:
   resolved "https://registry.yarnpkg.com/react-refresh/-/react-refresh-0.8.3.tgz#721d4657672d400c5e3c75d063c4a85fb2d5d68f"
   integrity sha512-X8jZHc7nCMjaCqoU+V2I0cOhNW+QMBwSUkeXnTi8IPe6zaRWfn60ZzvFDZqWPfmSJfjub7dDW1SP0jaHWLu/hg==
 
-react-router-dom@^5.1.2:
+react-router-dom@^5.3.0:
```

Comment:
Did you update the dependency on `package.json` as well?

Not sure if related, but the build is currently broken https://github.com/gnosis/gp-ui/runs/4483657518?check_suite_focus=true

Reply:
yes